YourToDo/templates/auth/registration.html

```diff
@@ -2,10 +2,46 @@
 
 {% block content %}
 	<div class="container">
-		<h1>Register</h1>
-		<form action="/registration/" method="post">{% csrf_token %}
-			{{form}}
-			<input type="submit" value="Register" />
-		</form>
+		<div class="form-registration">
+			<h2>Registration</h2>
+			<form action="/registration/" method="post">{% csrf_token %}
+				<div class="username">
+					<label for="{{ form.username.id_for_label }}">Username:</label>
+					{{ form.username }}
+					<p>Length: 30 characters or fewer. Letters, digits and '
+                    '@/./+/-/_ only.'<p>
+                    {{ form.username.errors }}
+				</div>
+				<div class="first_name">
```

Comment:
The main reason I did this was because of the password confirmation field. Wrapping these in their own divs allows easier editting for each label/input field since Django draws some of these entities on its own. You're right though, for some of them I can make a general div class as opposed to making a different class for each one